packages/popover-react/src/Popover.tsx

```diff
@@ -0,0 +1,268 @@
+import {
+    useFloating,
+    useFocus,
+    useHover,
+    useClick,
+    useDismiss,
+    autoUpdate,
+    useRole,
+    useMergeRefs,
+    offset,
+    flip,
+    shift,
+    useInteractions,
+    FloatingPortal,
+    FloatingFocusManager,
+    UseFloatingOptions,
+} from "@floating-ui/react";
+import { Props as FloatingFocusManagerOptions } from "@floating-ui/react/src/components/FloatingFocusManager";
+import { Props as ClickOptions } from "@floating-ui/react/src/hooks/useClick";
```

Comment:
For disse som henter fra hooks kan du bruke `Parameters` som er innebygget i TypeScript:

```suggestion
type ClickOptions = Parameters<typeof useClick>[0];
```

Bruker `[1]` her fordi innstillingene blir sendt inn som andre argument til funksjonen i form av et objekt, som er det vi er ute etter.